jib-core/src/main/java/com/google/cloud/tools/jib/builder/ProgressEventDispatcher.java

```diff
@@ -61,7 +64,7 @@
   public static ProgressEventDispatcher newRoot(
       EventDispatcher eventDispatcher, String description, long allocationUnits) {
     return newProgressEventDispatcher(
-        eventDispatcher, Allocation.newRoot(description, allocationUnits));
+        eventDispatcher, Allocation.newRoot(description, allocationUnits), null);
```

Comment:
maybe we should just not have a `Nullable` by adding a `BuildStepType` for the root, maybe called `ALL` or `COMPLETION` or something?